src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/CalculationResultEnrichmentService.kt

```diff
@@ -167,6 +141,11 @@ class CalculationResultEnrichmentService(
       val hints = mutableListOf<ReleaseDateHint>()
       if (calculationBreakdown?.breakdownByReleaseDateType?.containsKey(ReleaseDateType.HDCED) == true) {
         val hdcRules = calculationBreakdown.breakdownByReleaseDateType[ReleaseDateType.HDCED]!!.rules
+
```

Comment:
Used the presenence of rule HDCED_MINIMUM_CUSTODIAL_PERIOD from CRS-2038 to determine whether to  show the hint text